File: JsonInflatorAndroid/JsonInflatorLib/src/main/java/com/crescentflare/jsoninflator/binder/InflatorBinder.java
package com.crescentflare.jsoninflator.binder;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * Json inflator binder: default interface
 * An interface which can be implemented to do custom inflatable object binding
 */
@SuppressWarnings({"unused", "WeakerAccess"})
public interface InflatorBinder
{
    void onBind(@NotNull String refId, @Nullable Object object);
}

File: JsonInflatorIOS/JsonInflator/Classes/binder/InflatorBinder.swift
//
//  InflatorBinder.swift
//  Json inflator Pod
//
//  Library binder: default protocol
//  A protocol which can be implemented to do custom binding of objects during JSON inflation
//

public protocol InflatorBinder {
    
    func onBind(refId: String, object: Any)
    
}
